src/components/ClassOverview.vue

```diff
@@ -30,7 +30,7 @@
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: scopedName(property) } }"
-							class="pl-1 mr-2"
+							class="mx-2 mr-2"
```

Comment:
I may have explained this badly for someone now really knowing tailwindcss, sorry.

You need to remove `mr-2` here now that `mx-2` is a thing. `mx-2` is a shortcut for `ml-2 mr-2`

Reply:
My bad 🤦‍♂️